README.md

```diff
@@ -253,6 +255,42 @@ To delete a station, call the client method:
 data = client.delete_station('5ed2118acca8ce0001f1aeg1') # => nil
 ```
 
+#### Create Measurements
+
+To create measurements, call the client method:
+```ruby
+client.create_measurements([
+  {
+    "station_id": -1,
+    "dt": 1479817340,
+    "temperature": 18.7,
+    "wind_speed": 1.2,
+    "wind_gust": 3.4,
+    "pressure": 1021,
+    "humidity": 87,
+    "rain_1h": 2,
+    "clouds": [
+      {
+        "condition": 'NSC'
+      }
+    ]
+  }
+]) # => nil
+```
+
+#### Get Measurements
+
+To get measurements, call the client method with the mandatory parameters:
```

Comment:
mandatory -> required

Doesn't look like all these are required, so I'd say "The following parameters are required: ...".

Reply:
They are apparently all required, unfortunately >_< https://openweathermap.org/stations#get_measurements